cmake/setup_custom_targets.cmake

```diff
@@ -135,23 +56,19 @@ FILE(WRITE ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/print_info.cmake
 #                     and reruns the configure and generate phases of CMake
 #    rebuild_cache  - rerun the configure and generate phases of CMake
 #
-#    documentation  - ${_description_string} component 'documentation'
-#    examples       - ${_description_string} component 'examples'
-#    library        - ${_description_string} component 'library'
-#    package        - build binary package
```

Comment:
How will we build packages in the future? (I don't actually know what this target does, just asking.)

Reply:
I simply removed the description from the `make info` target. I will put everything back in place.